dask/array/tests/test_routines.py

```diff
@@ -1237,6 +1237,102 @@ def test_cov():
         da.cov(d, ddof=1.5)
 
 
+@pytest.mark.skipif(
+    not NUMPY_GE_220, reason="fweights is not an kwarg prior to numpy 2.2"
+)
+def test_cov_fweights():
+    x1 = da.array([[0, 2], [1, 1], [2, 0]]).T
+    res1 = da.array([[1.0, -1.0], [-1.0, 1.0]])
+    x2 = da.array([0.0, 1.0, 2.0], ndmin=2)
+    frequencies = da.array([1, 4, 1])
+    x2_repeats = da.array([[0.0], [1.0], [1.0], [1.0], [1.0], [2.0]]).T
+    res2 = da.array([[0.4, -0.4], [-0.4, 0.4]])
+    unit_frequencies = np.ones(3, dtype=np.int_)
+
+    assert allclose(da.cov(x2, fweights=frequencies), da.cov(x2_repeats))
```

Comment:
Would something like this be possible, to ensure we match NumPy?

```
result = da.cov(x2, fweights=frequencies)
expected = np.cov(x2.compute(), fweights=frequencies.compute())
assert_eq(result, expected)
```

Reply:
Yep, added this assertion. Thanks!